detekt.yml

```diff
@@ -539,7 +552,7 @@ style:
   UnusedImports:
     active: false
   UnusedPrivateClass:
-    active: false
```

Comment:
Why do we enable all these checks now?

Reply:
Because they were enabled in https://github.com/arturbosch/detekt/blob/master/detekt-cli/src/main/resources/default-detekt-config.yml
I merged our config with default config, so all new checks are imported correctly.